modules/cnf-image-based-upgrade-rollback.adoc

```diff
@@ -30,6 +30,7 @@ You can manually roll back the changes if you encounter unresolvable issues afte
 .Prerequisites
 
 * Log in to the hub cluster as a user with `cluster-admin` privileges.
+* Ensure that the control plane certificates on the original stateroot are valid. If the certificates expired, see _Recovering from expired control plane certificates_.
```

Comment:
```suggestion
* Ensure that the control plane certificates on the original stateroot are valid. If the certificates expired, see "Recovering from expired control plane certificates".
```
https://www.ibm.com/docs/en/ibm-style?topic=references-citations#general-guidelines

> If you refer to a smaller piece of work within a larger container such as a chapter within a book or an article in a magazine, use italic font for the larger container and double quotation marks for the smaller piece.

TL/DR double quote for referring to other docs sections